### DISCUSSION TITLE
opensans10.h broken

### DISCUSSION THREAD

**Discussion by holgerlembke** [vroland/epdiy#338] [General]
File is empty.

examples/weather/main/opensans10.h

**Comment by martinberlin** [comment]
Hi Holger,
converting this to a discussion. This is not any real issue that would avoid you to compile the example as it is not referenced in the main file:

```c
// fonts
#include "opensans12.h"
#include "opensans12b.h"
#include "opensans16.h"
#include "opensans16b.h"
#include "opensans24.h"
#include "opensans24b.h"
#include "opensans8.h"
#include "opensans8b.h"
```

**Comment by holgerlembke** [comment]
What to discuss?

The file is empty. So either delete it or fill it with valid content.

**Comment by martinberlin** [comment]
Nothing to discuss but is also not an Issue. We will clean this up in next iteration.
An issue is something that stops you to build something or a bug and this does not fall in any category.

>  The file is empty. So either delete it or fill it with valid content.

This is open source. Every user can also make Pull requests and collaborate with making this repository better. 
Holger this a collaborative project: We are not at your service. 


If you don't like something you can also make a Pull Request and fix it. It's not cool to open an Issue for every little thing you find. Open one if there is something serious, like a bug or real issue with the project.
If you want to collaborate you could also make your own "weather project" that compiles in IDF directly and is easier to test than this complicated Weather example that requires Arduino and other components to work.

**Comment by holgerlembke** [comment]
Martin, please.

I loosely follow the spirit of https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues with my usage of issues. I just use this as a tracker for things. In this case, I assume that it might be an error, that someone might have the file lying around and it could be fixed, or removed or whatever. 

I am in the process of moving the fonts into its own folder and make them a more universal thing. Therefor, look at https://github.com/holgerlembke/epdiy/tree/main/src/fonts. I have recreated opensans10.h , not sure it works...

There are 16 issues. 16. Once in a week there is another issue. Overwhelming. Not.

And for "We are not at your service.", have a look at https://github.com/holgerlembke/epdiy/tree/main/examples/Basics. Far for ready, still very chaotic, but at least some Arduino-IDE-look-alike stuff.

Please. No drama.

**Comment by martinberlin** [comment]
It's nothing personal and I'm not making a drama. Please add an Issue only if there is something blocking, a bug, or you want to propose an enhancement giving details of what you would like to achieve. 

> opensans10.h broken
 
It's not broken. It's just empty. For me a file that is empty does not fall in the Issue category if does not break the build. Instead of doing an Issue, use the same energy to make a fork, and propose a change directly.

> There are 16 issues. 16. Once in a week there is another issue. 

Before I started collaborating here there where more than 30 Issues and only half of them where real Issues, the rest where stale, or things nobody care about. Let's please keep issues just a few so we can focus in what is important.

CHECK: https://x.com/martinfasani/status/1813494915789136171
